tests/test-framework/src/bin/errcov.rs

```diff
@@ -1,241 +0,0 @@
-// // Copyright (C) 2019-2025 Provable Inc.
```

Comment:
Can we keep the `errcov` utility around in the new design?
It's outdated, but I think we do want to revive it at some point.

Reply:
Fair enough. I'll file an issue.